engines/bops_consultees/app/helpers/bops_consultees/application_helper.rb

```diff
@@ -12,5 +12,24 @@ def nav_items
     def home_path
       root_path
     end
+
+    def link_to_document(link_text, document, **args)
+      new_tab = /(new (window|tab)|<img\b)/.match?(link_text) ? "" : true
+
+      govuk_link_to(
+        link_text,
+        url_for_document(document),
+        new_tab:,
+        **args
+      )
+    end
+
+    def url_for_document(document)
+      if document.published?
+        main_app.api_v1_planning_application_document_url(document.planning_application, document)
```

Comment:
We now have a url endpoint in the consultees engine that might be more appropriate?

```suggestion
        planning_application_document_url(document.planning_application, document)
```

(actually, this should work whether published or not, so we may not need the conditional.)